### DISCUSSION TITLE
Save coordinates to shapefile

### DISCUSSION THREAD

**Discussion by Erl-gry** [opengisch/QField#2958] [Q&A]
Hi, I would like to save the coordinates of the captured points in the attribute table. However, I can't find any suitable variables that I can enter in the attribute form of the layer properties. 

If I load the layer into my QGis project after data collection, the points are displayed correctly in the project. An "info" on one of these points shows me the coordinates in the "derived" area of the result window and not in the attribute table. However, the data should be written directly into the table of the layer during data collection. What am I doing wrong?

For the positioning a Leica Zeno FLX100 is used.

Thank's for any help!

**Comment by VxTedxV** [comment]
Do it the same way you do it on QGis : on QGis, once you create a point on a layer, its coordinates aren't saved in the attribute table... 
You can use functions like this one for X coordinate, for example : https://docs.qgis.org/3.22/en/docs/user_manual/expressions/functions_list.html#x
